samples/micrometer-samples-core/src/main/java/io/micrometer/core/samples/ObservationHandlerSample.java

```diff
@@ -42,12 +42,17 @@ public static void main(String[] args) throws InterruptedException {
         Observation observation = Observation
                 .createNotStarted("sample.operation", new CustomContext(), observationRegistry)
                 .contextualName("CALL sampleOperation").keyValuesProvider(new CustomLocalKeyValuesProvider())
-                .lowCardinalityKeyValue("a", "1").highCardinalityKeyValue("time", Instant.now().toString()).start();
+                .lowCardinalityKeyValue("a", "1").highCardinalityKeyValue("time", Instant.now().toString()).longTask()
+                .start();
 
         try (Observation.Scope scope = observation.openScope()) {
             Thread.sleep(1_000);
             observation.error(new IOException("simulated"));
         }
+
+        System.out.println("---");
```

Comment:
Do we want to signal that this is before the observation stop?  i.e. `--- Before Stop ---`.   And for the System.out.println below have a similar signal `--- After Stop --- `.

Reply:
👍🏼